core/src/main/web/app/mainapp/components/notebook/outputdisplay/outputdisplayfactory-service.js

```diff
@@ -82,12 +82,27 @@
           });
         },
         link: function(scope, element, attrs) {
+          var tagstofilter = ['applet', 'base', 'basefont', 'body', 'frame', 'frameset', 'head', 'html',
```

Comment:
I did search for one when I did it for our sharing server and found nothing simple. There are some full sanitizers but these are very complex and work by whitelist - never updated since years so, for example, no svg.